### ISSUE TITLE
distorted image results

### ISSUE THREAD

**Issue by Gauravajariwal11** [DeokyunKim/Progressive-Face-Super-Resolution#20]
Hi sir, 

I am using your model to try to enhance the qualities of some faces that I cropped frame by frame from a video. Before testing it on my own dataset, I tried on some other pictures from celeba dataset, and for some reason the output is very bad. I have tried all different images like 16x16 and 128x128... the result is always distorted. Any idea what can be done here to improve?

or else, Is there any possibility that I can get your training code?

Thanks a lot. This paper is very good and looks very promising.

**Comment by DeokyunKim** [comment]
Hi, Gauravajariwal. 

As I mentioned in #19,  you should crop the face region in the image to 178x178 pixels. After that, resize 128x128, 64x64 and 16x16 pixel sequentially and it normalize it by mean [0.5, 0.5, 0.5] and std [0.5, 0.5, 0.5]. 

If the sample set distribution is different from training set, as a matter of course, up-scaled results are weird. Therefore, you should re-train the Face SR Model using your own dataset.

Although I can not upload fully training code, I will help you implementation of it.
If you have any questions of implementation training code, please comment that here.

Thanks.

**Comment by Gauravajariwal11** [comment]
Oh, I think the problem was that I did not resize it sequentially. But because my input data/picture is very much distorted and blurry, I think I will have to retrain the model and write implementation code. I will surely need your help if I do that.

Thanks.

**Comment by Gauravajariwal11** [comment]
hi,

Do you think I can get your email address and contact you there? 

Thanks.

**Comment by DeokyunKim** [comment]
deokyunkim@kaist.ac.kr
